### PR TITLE
fixed Using Amazon SageMaker Components link to point to right location

### PR DESCRIPTION
**Pull request by jsitu777**
**Description of your changes:**
Fixed related broken link Issue https://github.com/aws/amazon-sagemaker-operator-for-k8s/issues/197

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Comment by wenjinsitu** [comment]
@surajkota @rrrkharse

File: components/aws/sagemaker/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ With Amazon SageMaker Components for Kubeflow Pipelines (KFP), you can create an
 ## Components
 Amazon SageMaker Components for Kubeflow Pipelines offer an alternative to launching compute-intensive jobs in Kubernetes and integrate the orchestration benefits of Kubeflow Pipelines. The following Amazon SageMaker components have been created to integrate 6 key Amazon SageMaker features into your ML workflows. You can create a Kubeflow Pipeline built entirely using these components, or integrate individual components into your workflow as needed. 
 
-For an end-to-end tutorial using these components, see [Using Amazon SageMaker Components](https://sagemaker.readthedocs.io/en/stable/workflows/kubernetes/using_amazon_sagemaker_components.html).
+For an end-to-end tutorial using these components, see [Using Amazon SageMaker Components](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-components-for-kubeflow-pipelines.html).
```
Comment (surajkota) [inline components/aws/sagemaker/README.md:9]: one more link on this page https://github.com/kubeflow/pipelines/blob/cc8e3fdec14047802a062f9608177c92c5f9dba2/samples/contrib/aws-samples/mnist-kmeans-sagemaker/README.md

**Review by Arhell — COMMENTED** [review]
Please sign CLA

File: components/aws/sagemaker/batch_transform/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 Component to get inferences for an entire dataset in SageMaker from a Kubeflow Pipelines workflow.
 
 ## Details
-With [batch transform](https://docs.aws.amazon.com/sagemaker/latest/dg/how-it-works-batch.html), you create a batch transform job using a trained model and the dataset, which must be stored in Amazon S3. Use batch transform when you:
+With [batch transform](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-components-for-kubeflow-pipelines.html), you create a batch transform job using a trained model and the dataset, which must be stored in Amazon S3. Use batch transform when you:
```
Comment (surajkota) [inline components/aws/sagemaker/batch_transform/README.md:7]: this link should point to: https://docs.aws.amazon.com/sagemaker/latest/dg/batch-transform.html

**Comment by jsitu777** [comment]
@google-oss-prow I should have signed the CLA already. Am I good to go for future PRs?

File: samples/contrib/aws-samples/mnist-kmeans-sagemaker/README.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ You will also have an model endpoint in service. Refer to [Prediction section](#
   - Opening SageMaker [console](https://us-east-1.console.aws.amazon.com/sagemaker/home?region=us-east-1#/endpoints),  or
   - Clicking the `sagemaker-deploy-model-endpoint_name` under `Output artifacts` of `SageMaker - Deploy Model` component of the pipeline run
 
-2. Setup AWS credentials with `sagemaker:InvokeEndpoint` access. [Sample commands](https://sagemaker.readthedocs.io/en/stable/workflows/kubernetes/using_amazon_sagemaker_components.html#configure-permissions-to-run-predictions)
+2. Setup AWS credentials with `sagemaker:InvokeEndpoint` access. [Sample commands](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-components-tutorials.html)
```
Comment (surajkota) [inline samples/contrib/aws-samples/mnist-kmeans-sagemaker/README.md:48]: Refer to the `Configure permissions to run predictions` section in [AWS docs](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-components-tutorials.html) if you need sample commands

**Comment by surajkota** [comment]
@jsitu777 check if your email id is correct on the commits

**Comment by surajkota** [comment]
https://github.com/kubeflow/pipelines/pull/8035/checks?check_run_id=7364658211

**Pull request closed without merging**